source/backend/apimodels/Models/Concepts/CompensationRequisition/CompensationRequisitionModel.cs

```diff
@@ -72,6 +74,10 @@ public class CompensationRequisitionModel : BaseAuditModel
 
         public ProjectModel AlternateProject { get; set; }
 
-        public IEnumerable<CompensationRequisitionPropertyModel> CompensationRequisitionProperties { get; set; }
+        public IEnumerable<CompReqLeaseStakeholderModel> CompReqLeaseStakeholder { get; set; }
+
+        public IEnumerable<CompReqAcquisitionPropertyModel> CompReqAcquisitionProperties { get; set; }
```

Comment:
Hmm, why can't these be FilePropertyModels?

Reply:
que need the ID of the parent, not just the property entity since the parent is the one keeping track of the relationship to the comp req.